docs/source/pages/guides/raspberrypi.rst

```diff
@@ -0,0 +1,91 @@
+OAK & Raspberry Pi
+==================
+
+Our devices can also be connected to small SBCs like a Raspberry Pi (RPi). To install `depthai <https://github.com/luxonis/depthai-python>`__
+on a RPi, you should install `dependencies <https://docs.luxonis.com/projects/api/en/latest/install/#raspberry-pi-os>`__
+and then install the `library <https://docs.luxonis.com/projects/api/en/latest/install/#install-from-pypi>`__ with :code:`pip`.
+Depthai library has **prebuilt wheels** for RPi on `piwheels <https://www.piwheels.org/project/depthai/>`__.
+
+Using pre-configured RPi OS image
+*********************************
+
+We provide Raspberry Pi OS images that have **depthai and all its dependencies pre-installed**. **Images are** `available here <https://drive.google.com/drive/folders/1O50jPpGj_82jkAokdrsG--k9OBQfMXK5?usp=sharing>`__.
+
+- **KS_Rpi_OS_Full** - Uses RPi Full OS, has pre-installed depthai and `CEP <https://github.com/cortictechnology/cep>`__. **Required hardware**: 32 GB uSD card or larger, RPi 4 with 4 GB RAM. This image was flashed on uSD cards that were sent to KickStarter backers
+- **OAK_CM4_POE** - Uses RPi OS (default one), has pre-installed depthai. **Required hardware**: 8 GB uSD card or larger. This image is flashed on uSD cards that get sent with `OAK-D-CM3 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__, `OAK-D-CM4 <https://docs.luxonis.com/projects/hardware/en/latest/pages/DM1097.html>`__, `OAK-D CM4 PoE <https://docs.luxonis.com/projects/hardware/en/latest/pages/DM2097.html>`__
```

Comment:
@Luxonis-David  is this true? Do we use this img for all these devices?

Reply:
True, we will do so for all devices going forward, though we never flashed the CM3 with this image.
I would need to try if it works on CM3, but IIRC Sachin already tested this and it works.